extensions/wikia/Discussions/DiscussionsHooksHelper.php

```diff
@@ -73,6 +73,20 @@ public static function onLinkBegin( $skin, $target, &$text, &$customAttribs, &$q
 		return true;
 	}
 
+	public static function onBeforePageDisplay( \OutputPage $out ): bool {
+		global $wgEnableForumMigrationMessage, $wgEnableForumMigrationMessageGlobal, $wgEnableForumExt;
+
+		$enabledGlobally =
+			WikiFactory::getVarValueByName( 'wgEnableForumMigrationMessageGlobal', WikiFactory::COMMUNITY_CENTRAL )
+				?: $wgEnableForumMigrationMessageGlobal;
+
+		if ( $wgEnableForumMigrationMessage && $enabledGlobally && $wgEnableForumExt ) {
```

Comment:
We should return early if forum is not enabled (via `$wgEnableForumExt`) to avoid redundant calls to WikiFactory.

Reply:
good catch! 🙇